doc/man7/migration_guide.pod

```diff
@@ -467,6 +467,17 @@ Remove your usage of the low level APIs. In this case you will need to rewrite y
 
 =back
 
+=head3 Error code changes
+
+As OpenSSL 3.0 provides a brand new Encoder/Decoder mechanism for working with
+widely used file formats, the application authors should consider changes the checks
```

Comment:
`an application code that checks for particular error reason codes on key loading failures might need an update.`

Reply:
Fixed. Thanks!